tetrateci/create_istio_release.sh

```diff
@@ -71,6 +71,12 @@ cp -r ../istio .
 # export IMAGE_VERSION=$(curl https://raw.githubusercontent.com/istio/test-infra/master/prow/config/jobs/release-builder.yaml | grep "image: gcr.io" | head -n 1 | cut -d: -f3)
 # make shell TODO: https://github.com/tetratelabs/getistio/issues/82
 
+# Enable CGO flag"
+
+if [[ ${TAG} =~ "fips" ]]; then
+  text="if [[ "\${GOARCH}" == "amd64" ]]; then export CGO_ENABLED=1; else export CGO_ENABLED=0; fi"
```

Comment:
Add a echo line before this so that we know the script is `enabling CGO`?

`echo "Enabling CGO for FIPS build via CGO_ENABLED=1 to istio/common/scripts/gobuild.sh"`